fineract-savings/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccountTransaction.java

```diff
@@ -337,6 +338,15 @@ public static SavingsAccountTransaction releaseAmount(SavingsAccountTransaction
                 accountTransaction.refNo);
     }
 
+    public static SavingsAccountTransaction accrual(final SavingsAccount savingsAccount, final Office office, final LocalDate date,
+            final Money amount, final boolean isManualTransaction) {
+        final boolean isReversed = false;
+        final Boolean lienTransaction = false;
+        final String refNo = ExternalId.generate().getValue();
+        return new SavingsAccountTransaction(savingsAccount, office, SavingsAccountTransactionType.ACCRUAL.getValue(), date, amount,
```

Comment:
I think balance portions are important: fee, penalty and interest portion of the transaction.